rep-2004.rst

```diff
@@ -13,18 +13,41 @@ Abstract
 
 This REP describes a set of categories meant to convey the quality or maturity of packages in the ROS ecosystem.
 Inclusion in a category, or quality level, is based on the policies to which a package adheres.
-The categories address version, change control, documentation, testing, dependency and platform support policies.
+The categories address policies about versioning, change control, documentation, testing, dependencies, platform support and security.
 
 Motivation
 ==========
 
-The purpose of this REP is to provide standard guidelines regarding package quality, for both maintainers and consumers.
+The purpose of this REP is to provide standard guidelines for evaluating package quality, for both maintainers and consumers.
 
-The categories are meant to set quality expectations for package consumers, not to enforce quality rules on maintainers.
-Package maintainers are responsible for establishing processes and documenting how their package's policies achieve a certain quality level.
-With the help of documented policies, package consumers can better evaluate whether a package or its dependencies meet the standards for use in their projects.
+Package maintainers can use these guidelines to establish and document their own specific policies addressing how their packages achieve certain quality levels.
 
-By providing these rough goals for package maintainers to strive towards, the categories will also encourage better quality across the ROS ecosystem.
+Consumers can use the guidelines in the REP, as well as the corresponding policies defined by maintainers, to set expectations on package quality and better evaluate whether a package or its dependencies meet the standards of use in their projects.
+
+By setting these goals and expectations for maintainers and consumers, respectively, the guidelines will also encourage better quality across the ROS ecosystem.
```

Comment:
I think the last two sentences from the antigoal section belong here (feel free to ignore this one if you think they're better off below, I just think since they go back to talking about goals they shouldn't be in a section with an opposite purpose)

```suggestion
The outcome of this REP should be that maintainers who want to evaluate their packages' quality have looked at each guideline, thought about how their practices address each, adjusted their policies if needed, and then documented their policies and justifications. 

In turn, the documentation of policies and communication of quality characteristics between maintainers and consumers will improve.
By setting these goals and expectations for maintainers and consumers, respectively, the guidelines will also encourage better quality across the ROS ecosystem.
```